doas.c

```diff
@@ -189,6 +217,9 @@ checkconfig(const char *confpath, int argc, char **argv,
 		err(1, "setresuid");
 
 	parseconfig(confpath, 0);
+#ifdef DOAS_CONFDIR
+	parseconfdir(DOAS_CONFDIR, 0);
```

Comment:
Wasn't the idea to support either `doas.conf` or `doas.d`, to avoid having to think which one goes first or anything?

And this should use `confpath`, not the macro, right?

Reply:
It’s very common to have the main configuration file that includes configuration snippets from a directory. In this case, the “include” is implicit, but the same logic applies – /etc/doas.conf is loaded first, then /etc/doas.d/* (if exists) in chronological order. I think that it’s more conventional and intuitive than loading either doas.conf, or doas.d/*.

Reply:
https://github.com/Duncaen/OpenDoas/issues/61#issuecomment-892025672 seems to point out a preference for one or the other though...

Reply:
I don't mind changing it to be either confdir or static config file, if that's what we want to do.  That also makes a lot of sense.

Then end users would just add their own snippets to `/etc/doas.d`.

Reply:
Resolved by making it one or the other.